holmes/config.py

```
@@ -86,6 +86,69 @@ def _toolset_tools_changed(
     return False
 
 
+def load_secrets_from_files(secret_dir: str = "/etc/holmes/secrets") -> dict:
```

Ruff (0.15.18)

[error] Possible hardcoded password assigned to function default: "secret_dir"

(S107)